PIM-9084: Block the export of attributes with specific locale wich isn't the used locale in the export profile (#12306)
akeneo/pim-community-dev: CHANGELOG-3.2.md

# 3.2.x

## Bug fixes

> - PIM-9084: Block the export of attributes with specific locale wich isn't the used locale in the export profile

Reviewer: ```suggestion
- PIM-9084: Filter locale specific attributes in exports when the value's locale and the export profile's locale are different
```
